apis/v1alpha1/gateway_types.go

```diff
@@ -418,7 +418,7 @@ type RouteBindingSelector struct {
 	// application protocol specified in the Listener's Protocol field.
 	//
 	// If an implementation does not support or recognize this
-	// resource type, it SHOULD raise a "ConditionInvalidRoutes"
+	// resource type, it SHOULD raise a "InvalidRoutesRef"
```

Comment:
This actually is a bit more complex than this. Maybe something like:

```
SHOULD set the "ResolvedRefs" condition to false for this listener with the "InvalidRoutesRef" reason.
```

More context here: https://github.com/kubernetes-sigs/service-apis/blob/master/apis/v1alpha1/gateway_types.go#L752-L782

/cc @jpeach

Reply:
Thank you! Updated.